MANANPAT-M-VJMD:backup_nh mananpat$ git diff ../../../../../internal/cisco/flags/flags.go
diff --git a/internal/cisco/flags/flags.go b/internal/cisco/flags/flags.go
index 6d61d8b9..dee22435 100644
--- a/internal/cisco/flags/flags.go
+++ b/internal/cisco/flags/flags.go
@@ -14,7 +14,7 @@ var (
        GRIBIAFTChainCheck = flag.Bool("gribi_aft_chain_check", false,
                "This enable/disable AFT chain check for gribi prefix in gribi tests.")
 
-       GRIBIFIBCheck = flag.Bool("gribi_fib_check", false,
+       GRIBIFIBCheck = flag.Bool("gribi_fib_check", true,
                "This enable/disable FIB ack check for gribi entries in gribi tests.")
 
        GRIBIRIBCheck = flag.Bool("gribi_rib_check", true,